src/admin-views/widgets/components/dropdown.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Admin View: Widget Dropdown Component
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/admin-views/widgets/components/dropdown.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://m.tri.be/1aiy
+ *
+ * @var string  $label   Label for the checkbox.
+ * @var string  $value   Value for the checkbox.
+ * @var string  $id      ID of the checkbox.
+ * @var string  $name    Name attribute for the checkbox.
+ * @var boolean $checked Whether the checkbox is checked or not.
+ *
+ * @version TBD
+ *
+ */
+
+?>
+<p
+		class="tribe-filter-bar-c-dropdown tribe-common-form-control-dropdown"
+>
+	<label
+			class="tribe-common-form-control-dropdown__label"
+			for="<?php echo esc_attr( $id ); ?>"
+	>
+		<?php echo esc_html( $label ); ?>
+	</label>
+	<select
+			id="<?php echo esc_attr( $id ); ?>"
+			name="<?php echo esc_attr( $name ); ?>"
+			class="tribe-common-form-control-dropdown__input widefat"
+	>
+		<?php foreach ( $options as $option ) {
+			?>
+			<option
+					value="<?php echo $option['value']; ?>"
+					<?php echo $option['value'] == $value ? 'selected="selected"' : ''; ?>
+					<?php // todo why does checked not work? it shows as selected? can I typecast? but what about supporting string and int? ?>
+					<?php //checked( $option['value'], $value ); ?>
+			>
```

Comment:
* :warning: This comment is 70% valid code; is this commented out code?
* :no_entry_sign: No space found before comment text; expected "// checked( $option['value'], $value );" but found "//checked( $option['value'], $value );"
* :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 05d4f97b6f5f30591430e76a7590bf42</sub>